README.md

```diff
@@ -68,93 +68,132 @@ services:
+    image: hashicorp/http-echo
     ports:
     - mode: ingress
       target: 5678
       published: 5678
       protocol: tcp
 ```
 
-You can then use that Compose file like any other. You could save it to disk or pipe it straight to `docker stack` or `docker-compose` to launch the application.
+You can then use that Compose file like any other. You could save it to disk or
+pipe it straight to `docker stack` or `docker-compose` to launch the
+application.
 
-```
-$ docker-app render | docker-compose -f - up
+```console
+$ docker app render | docker-compose -f - up
 ```
 
-This is where it gets interesting. We can override those parameters at runtime, using the `--set` option. Let's specify different option and run `render` again:
+This is where it gets interesting. We can override those parameters at runtime,
+using the `--set` option. Let's specify different option and run `render` again:
 
-```
-$ docker-app render --set version=0.2.3 --set port=4567 --set text="hello production"
+```console
+$ docker app render --set port=4567 --set text="hello production"
 version: "3.2"
 services:
   hello:
     command:
     - -text
     - hello production
-    image: hashicorp/http-echo:0.2.3
+    image: hashicorp/http-echo
     ports:
     - mode: ingress
       target: 5678
       published: 4567
       protocol: tcp
 ```
 
-If you prefer you can create a standalone configuration file to store those parameters. Let's create `prod.yml` with the following contents:
+If you prefer you can create a standalone configuration file to store those
+parameters. Let's create `prod.yml` with the following contents:
 
 ```yaml
-version: 0.2.3
 text: hello production
 port: 4567
 ```
 
 You can then run using that configuration file like so:
 
+```console
+$ docker app render --parameters-file prod.yml
 ```
-$ docker-app render -f prod.yml
+
+You can share your application package by pushing it to a container registry.
+
+```console
+$ docker app push --tag myrepo/hello:0.1.0
+```
+
+Others can then use your application by specifying the registry tag.
+
+```console
+$ docker app inspect myrepo/hello:0.1.0
 ```
 
 More examples are available in the [examples](examples) directory.
 
 ## CNAB
 
-Under the hood `docker-app` is CNAB compliant. It generates a CNAB from your application source and is able to install and manage any other CNAB too.
-CNAB specifies three actions which `docker-app` provides as commands:
-- `install`
-- `upgrade`
-- `uninstall`
+Under the hood `docker app` is CNAB compliant. It generates a CNAB from your
+application source and is able to install and manage any other CNAB too. CNAB
+specifies three actions which `docker app` provides as commands:
+* `install`
+* `upgrade`
+* `uninstall`
 
-**Note**: These commands need a Docker Context so that `docker-app` knows which endpoint and orchestrator to target.
+**Note**: These commands need a Docker Context so that `docker app` knows which
+Docker endpoint to target.
```

Comment:
What about adding some material explaining the difference between the context to execute the invocation image and the target context?

Reply:
Yep, and maybe state that in the future, we'll probably support `kubeconfig` files for kubernetes workloads as well

Reply:
That is not strictly true anymore: default target context is the same as current active context.
Defining a target context is only required for installing on a remote cluester

Reply:
I actually think that this shouldn't be under the CNAB section. I'll remove it.

Reply:
The context part should be in another example. We don't need it when we're showing CNAB compliance.